main.pm

```diff
@@ -40,14 +40,22 @@ sub load_osautoinst_tests() {
 }
 
 sub load_openQA_tests() {
-    loadtest "openQA/dashboard.pm";
-    loadtest "openQA/login.pm";
-    return 1 if get_var('INSTALL_ONLY');
-    loadtest "openQA/build_results.pm";
-    loadtest "openQA/test_live.pm";
-    loadtest "openQA/test_results.pm";
-    loadtest "openQA/tests.pm";
-    loadtest "openQA/admin.pm";
+    if (get_var("OPENQA_CONTAINERS")) {
```

Comment:
I think an early return might be nice here, but no strong opinion

Reply:
we have an else branch. I don't see an early-return helping